test/unit_test.vcxproj

```diff
@@ -167,6 +167,7 @@
   <ItemGroup>
     <ClCompile Include="block_iterator_test.cpp" />
     <ClCompile Include="i_node_test.cpp" />
+    <ClCompile Include="jack_move_test.cpp" />
```

Comment:
This line tells Visual Studio to compile that source file as part of the unit test program